src/test/request_cancellation_test.cc

```diff
@@ -175,7 +177,7 @@ class RequestCancellationTest : public ::testing::Test {
 };
 
 TRITONSERVER_Server* RequestCancellationTest::server_ = nullptr;
-std::vector<int32_t> RequestCancellationTest::input0_data_(16, 1);
+std::vector<int32_t> RequestCancellationTest::input0_data_(1000, 1);
```

Comment:
Q: What's the point of changing it here?

Reply:
It has to match the `    std::vector<int64_t> input0_shape({1, 1000});` , which is also the shape specified in the `config.pbtxt` of the model being used for testing